### DISCUSSION TITLE
Home-built sewing machine

### DISCUSSION THREAD

**Discussion by tyrosinase** [inkstitch/inkstitch#1740] [General]
Someday I want to build an embroidery machine from scratch.

https://hackaday.com/2022/07/17/a-home-made-sewing-machine-may-be-the-only-one/

**Comment by X3msnake** [comment]
something like this is the best approach to a intial solution

https://inkstitch.org/tutorials/embroidery-machine/

**Comment by tyrosinase** [comment]
Exactly, but without starting with the treadle machine (as I've joked about in the past).

**Comment by X3msnake** [comment]
well, do it :)
and share the plans :)

No dia domingo, 17 de julho de 2022, Tyrosinase ***@***.***>
escreveu:

> Exactly, but without starting with the treadle machine (as I've joked
> about in the past).
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/inkstitch/inkstitch/discussions/1740#discussioncomment-3166328>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACUR56RMDND6GBSB4SQOEWTVUQ5W7ANCNFSM53Z6REXQ>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
Com os melhores cumprimentos,
Vinicius Silva
